edge-patron/src/main/resources/domain/edge-patron/patron-junit.feature

```diff
@@ -0,0 +1,38 @@
+Feature: edge-patron integration tests
+
+  Background:
+    * url baseUrl
+    * table modules
+      | name                      |
+      | 'okapi'                   |
+      | 'mod-login'               |
+      | 'mod-permissions'         |
+      | 'mod-inventory'           |
+      | 'mod-inventory-storage'   |
+      | 'mod-feesfines'            |
+
+    * table adminAdditionalPermissions
+      | name |
+
+    * table userPermissions
```

Comment:
What is the difference between admin permission and user permissions? Do the tests use two separate users?